dev/com.ibm.ws.jaxws.2.3.common/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -15,7 +15,7 @@
     <OCD id="com.ibm.ws.jaxws.clientConfig.metatype" description="%webserviceClient.desc" name="%webserviceClient" ibm:alias="webServiceClient" ibm:beta="true" >
          <AD id="serviceName" name="%serviceName" description="%serviceName.desc" required="false" type="String" />
          <AD id="ignoreUnexpectedElements" name="%ignoreUnexpectedElements" description="%ignoreUnexpectedElements.desc" required="false" type="Boolean" default="false" />
-         <AD id="enableSchemaValidation" name="%enableSchemaValidation" description="%enableSchemaValidation.desc" required="false" type="Boolean" default="true" />
+         <AD id="enableSchemaValidation" name="%enableSchemaValidation" description="%enableSchemaValidation.desc" required="false" type="Boolean"/>
```

Comment:
Is there any specific reason why default value is removed?

Reply:
Yep, I put it in the PR's first comment, but here it is:

> Removes the default enableSchemaValidation value of true to prevent LibertyWebServiceClientInInterceptor from applying the configuration, as this will always remove CXF's DefaultValidationEventHandler

Reply:
Unless they specially want more detailed schema validation, we don't want to change the behavior since it will change the exceptions messages.